### DISCUSSION TITLE
mastercontainer fails to pull images

### DISCUSSION THREAD

**Discussion by tfedder-de** [nextcloud/all-in-one#3912] [Questions] [answered]
### Steps to reproduce
1. -

### Expected behavior 
The mastercontainer should pull the images for the managed containers automatically and start containers from the new image after the update.

### Actual behavior 
Today (2023-12-11) after the daily backup, not all containers started and therefore the Nextcloud wasn't available.
This time the images for watchtower and notify-push could not be pulled by the mastercontainer automatically.
That happend to me on Friday (2023-12-08) as well, then with the collabora image.
In both cases running the command suggested in the log did pull the images and the containers could be started via the AIO interface afterwards. Although, as you can see in the log below, the image names are displayed URL escaped, so I needed to replace it with an actual slash and colon.

Unfortunately I did not have the time to look into DockerActionManager.php, but I will in case I encounter that error again.

### Host OS 
Debian 10

#### Nextcloud AIO version 
7.7.1

#### Current channel 
stable

#### Other valuable info 
```
nextcloud-aio-mastercontainer  | Fatal error: Uncaught Exception: Could not pull image nextcloud%2Faio-watchtower%3Alatest. Please run 'sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud%2Faio-watchtower%3Alatest' in order to find out why it failed. in /var/www/docker-aio/php/src/Docker/DockerActionManager.php:589
nextcloud-aio-mastercontainer  | Fatal error: Uncaught Exception: Could not pull image nextcloud%2Faio-notify-push%3Alatest. Please run 'sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud%2Faio-notify-push%3Alatest' in order to find out why it failed. in /var/www/docker-aio/php/src/Docker/DockerActionManager.php:589
```

**Comment by szaimen** [comment, marked as answer]
> Please run 'sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud/aio-notify-push:latest' in order to find out why it failed.
